test/uk/gov/hmrc/cdsimportsddsfrontend/controllers/model/DeclarationViewModelSpec.scala

```diff
@@ -51,7 +51,7 @@ class DeclarationViewModelSpec extends WordSpec with Matchers {
         List(Classification(Some("76071111"), Some("TSP")), Classification(Some("10"), Some("TRC")),
           Classification(Some("1234"), Some("TRA")), Classification(Some("VATZ"), Some("GN"))),
         Some(GoodsMeasure(Some("50"), Some("100"), Some("60"))),
-        Some(DutyTaxFee(Some("100"), Some(Payment("E")))),
+        Some(DutyTaxFee(Some("100"), None, Some(Payment("E")))),
```

Comment:
can we populate the viewmodel with some test data and assert this gets into the domain model, instead of relying on viewmodel defaults here?

Reply:
@ellamdav - yes, done.